code/modules/vehicle/sportscar.dm

```diff
@@ -0,0 +1,47 @@
+/obj/vehicle/car
+	name = "sports car"
+	desc = "A very luxurious vehicle."
+	icon = 'icons/vehicles/sportscar.dmi'
+	icon_state = "sportscar"
+	generic_pixel_x = 0
+	generic_pixel_y = 4
+	vehicle_move_delay = 1
+	var/static/image/carcover = null
```

Comment:
Basically, BYOND's static keyword works surprisingly similar to how it does in other languages such as C++, C#, Java, etc.